From 26d1fceba2a7677aaaed9968ec57a4dae970ca9b Mon Sep 17 00:00:00 2001
From: Jiangli Yuan <jlyuan@motorola.com>
Date: Thu, 14 Jan 2016 15:13:13 -0800
Subject: [PATCH 617/959] Revert "IKSWM-13107:8952/defconfig: Enable eMMC cmd
 queue kernel option"

 This reverts enabling of CONFIG_MMC_CQ_HCI.

 We see stability issue of cmdq under stress IO testing.

Change-Id: Ica8ad9f5cac29f72cfcae436fc8bd4328e9dea74
Reviewed-on: http://gerrit.mot.com/813552
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Submit-Approved: Jira Key <jirakey@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Zhi-Ming Yuan <a14194@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index c3aa421ba83..de57cc5e772 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -59,4 +59,3 @@ CONFIG_MMC_FFU=y
 CONFIG_SUSPEND_DEBUG=y
 CONFIG_ALSA_TO_H2W=y
 CONFIG_ESD_FS=y
-CONFIG_MMC_CQ_HCI=y
-- 
2.11.0

